src/framework/preferences/js/PrefsEditor.js

```diff
@@ -380,7 +380,7 @@ var fluid_2_0 = fluid_2_0 || {};
 
         var initialModel = that.initialModel,
             userSelections = fluid.copy(that.model),
-            changedSelections = {};
+            changedPrefs = {};
```

Comment:
I see that the userSelections has everything. I think the names are still confusing though. Maybe we should change userSelections to modelToSave or something. As there may be other things in there than user Selection which were analogous to preference selections before.